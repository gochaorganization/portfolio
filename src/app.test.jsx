import React from 'react'
import { shallow } from 'enzyme'
import App from './app'

describe('App tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})


    // expect(wrapper.find('.footer__list').exists()).toBeTruthy()

/*
1. Snapshot
2. Testing props. 
3. Testing data types (optional) - jest-extended 
4. Event Testing
5. Testing state

6.Check test coverage
*/
