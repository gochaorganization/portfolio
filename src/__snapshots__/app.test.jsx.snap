// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests renders correctly 1`] = `ShallowWrapper {}`;
